### PR TITLE
Fixed Issue #36: Added Drop down for past Events

### PR DESCRIPTION
**Pull request by chidelmun**
Issue Fixed.

### CODE REVIEW

**Comment by xSavitar** [comment]
Thanks for your PR. Will review and merge soon!

File: index.html [hunk 1]
```diff
@@ -56,21 +56,26 @@
...
                     <li><a data-scroll href="#faq" title="Questions about SMConf 2016 - Silicon Mountain Conference">FAQ</a></li>
-                    <li><a data-scroll href="#photos" title="Photos of SMConf 2015 - Silicon Mountain Conference">2015</a></li>
-		            <li><a data-scroll href="#location" title="Chariot Hotel Buea, the Venue of SMConf 2016 - Silicon Mountain Conference">Venue</a></li>
-                
+                    <li id="pst"><a data-scroll href="#" title="Photos of SMConf 2015 - Silicon Mountain Conference">Past Events</a>
+                    <ul id="dropdown">
+                        <li> <a href="#"> 2016 </a></li>
+                        <li> <a href="#photos"> 2015 </a></li>
+                        <li> <a href="#"> 2014 </a></li>
+                        <li> <a href="#"> 2013 </a></li>
```
Comment (awasum) [inline index.html:71]: There was no 2013 or 2014 event.

Comment (chidelmun) [reply index.html:71]: Sure there was one at Activspaces in 2013 and 2014 not big though. That's when the brainstorming all started till we organized the first big smconf in 2015 at Etta Palace.

**Comment by awasum** [comment]
Ok

**Comment by chidelmun** [comment]
My PRs will go live automatically on Heroku: Test them here https://smconf.herokuapp.com Mirrored from  https://git.heroku.com/smconf.git

**Comment by Izakey** [comment]
The first SMConf took place in 2015.

**Comment by chidelmun** [comment]
I  will need commit access soon @Izakey  @Ch3ck .  It's annoying  to have your code pending for review when people who are not committing have that right.

File: index.html (hunk 1 above)
Comment (Izakey) [reply index.html:71]: The Maiden ( First ) Silicon Mountain Conference was SmConf 2015 which took place on June 20th in Eta Palace Hotel.

**Pull request merged**